packages/ui-components/src/__tests__/WalletProvider.test.ts

```
@@ -0,0 +1,49 @@
+			}
+		});
+
+		expect(vi.mocked(setContext)).toHaveBeenCalledWith(USE_ACCOUNT_KEY, expect.any(Function));
+	});
+
+	it('should use default null account when no account prop provided', () => {
+		render(WalletProvider);
+
+		const setContextCalls = vi.mocked(setContext).mock.calls;
+		const accountCall = setContextCalls.find((call) => call[0] === ACCOUNT_KEY);
+		const defaultAccount = accountCall![1];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)